ldapauthenticator/ldapauthenticator.py

```diff
@@ -62,6 +63,9 @@ def _server_port_default(self):
             	uid={username},ou=people,dc=wikimedia,dc=org,
             	uid={username},ou=Developers,dc=wikimedia,dc=org
         	]
+
+        Active Directory Example:
+            DOMAIN\{login}
```

Comment:
I'm curious why this is needed? I'm on AD and I login with my samid (`DOMAIN\samid`) which is used to look up my CN which is then substituted into my DN template.